### PR TITLE
1456: Styled Recently added video block

### PR DESCRIPTION
**Pull request by Gravdan**

### CODE REVIEW

File: web/themes/custom/tube_theme/scss/misc.scss [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  @include open-sans-font(600);
+  @include font-size(14px, 19px);
+  text-align: left;
+  text-transform: uppercase;
+  color: $light-gray;
+}
+.row {
+  flex-wrap: nowrap;
+}
+.container-fluid {
```
Comment (ashabed) [inline web/themes/custom/tube_theme/scss/misc.scss]: The row and container-fluid classes are core parts of the bootstrap framework, we shouldn't change how they fundamentally work. I'll merge this PR but undo that change

File: web/themes/custom/tube_theme/templates/block--views-block--recently-added-videos-block-1-2.html.twig [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    'block-' ~ configuration.provider|clean_class,
+    'block-' ~ plugin_id|clean_class,
+  ]
+%}
+{%
+  set title_classes = [
+     'block__title',
+  ]
+%}
+<div{{ attributes.addClass('recent') }}>
```
Comment (ashabed) [inline web/themes/custom/tube_theme/templates/block--views-block--recently-added-videos-block-1-2.html.twig]: Adding the recent class is fine, but FYI all blocks are already assigned a class/ID that uniquely identify them, so it is not strictly necessary.

File: web/themes/custom/tube_theme/templates/page.html.twig [hunk 3]
```diff
@@ -0,0 +1,190 @@
+{#
```
Comment (ashabed) [inline web/themes/custom/tube_theme/templates/page.html.twig]: Were any changes made in this file? Generally speaking, when creating a new twig file (based on an existing one), you should commit the 'creation' of the file in one commit. Then you can commit edits to the file in subsequent commits. As-is, I can't tell what changes were made to the file, which makes it difficult to review

File: web/themes/custom/tube_theme/templates/page.html.twig [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+        <div class="featured-top">
+          <aside class="featured-top__inner section {{ container }} clearfix" role="complementary">
+            {{ page.featured_top }}
+          </aside>
+        </div>
+      {% endblock %}
+    {% endif %}
+    <div id="main-wrapper" class="layout-main-wrapper clearfix">
+      {% block content %}
+        <div id="main" class="{{ container-fluid }}">
```
Comment (ashabed) [inline web/themes/custom/tube_theme/templates/page.html.twig]: Is there a variable named container-fluid? If you want the class to be set as "container-fluid", you can do so directly without using a variable:
class="container-fluid"

I think you were overriding the default variable which says: `class="{{ container }}"` . That variable typically comes from a setting in the theme, so you could edit the theme's settings from the UI and change the default container to container-fluid, which would update the variable's value to container-fluid.

File: web/themes/custom/tube_theme/templates/page.html.twig (hunk 3 above)
Comment (Gravdan) [reply web/themes/custom/tube_theme/templates/page.html.twig]: Thank you for feedback. Page.html.twig file I only changed to container-fluid on the content area to make it full width. That is the only changes I made.

File: web/themes/custom/tube_theme/templates/page.html.twig (hunk 4 above)
Comment (Gravdan) [reply web/themes/custom/tube_theme/templates/page.html.twig]: I did not check if there was any variable named container-fluid. I read Bootstrap by default is set to container, fixed width and I could change it to container-fluid in page.html.twig file. What would be the best way to make it full width?

File: web/themes/custom/tube_theme/templates/page.html.twig (hunk 3 above)
Comment (Gravdan) [reply web/themes/custom/tube_theme/templates/page.html.twig]: Thank you letting me know. I’ll commit the file seprately next time.

**Pull request merged**